ansible/roles/vmware-template/tasks/create.yml

```diff
@@ -90,15 +70,19 @@
 
 - name: Add additional disks
   when: vm_hdd is defined
-  script: >
-    add_disk_to_vm.py
-    --host '{{ vcenter_hostname }}'
-    --user '{{ vcenter_username }}'
-    --password '{{ vcenter_password }}'
-    --uuid '{{ template_vm_uuid }}'
-    --disk-type '{{ item.type }}'
-    --disk-size '{{ item.size }}'
-  with_items: "{{ vm_hdd }}"
+  vmware_guest:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: "{{ vcenter_validate_certs }}"
+    datacenter: "{{ vcenter_datacenter }}"
+    folder: "{{ template_folder }}"
+    name: "{{ template_name }}"
+    disk:
```

Comment:
Please try to get list of disks from vmware_guest_facts and append new additional disks to it.